renderer/src/components/ActivityLog.tsx

```diff
@@ -9,7 +9,7 @@ const dateTimeFormat = new Intl.DateTimeFormat(window.navigator.language, {
 })
 
 const ActivityLogItem: FC<Activity> = (activity) => {
-  const time = dateTimeFormat.format(new Date(activity.timestamp))
+  const time = dateTimeFormat.format(new Date(activity.date))
```

Comment:
Adjusted to use `core`'s data schema. Or should we change `core` to match the previous schema here?

Reply:
Good point! https://github.com/filecoin-station/core/pull/95